go.mod

```diff
@@ -1,15 +1,71 @@
 module tls-secret-injector
 
-go 1.16
+go 1.17
 
 require (
 	github.com/sirupsen/logrus v1.8.1
-	github.com/spf13/cobra v1.1.1
+	github.com/spf13/cobra v1.3.0
 	github.com/spf13/pflag v1.0.5
-	github.com/spf13/viper v1.7.0
+	github.com/spf13/viper v1.10.1
 	github.com/stretchr/testify v1.7.0
-	k8s.io/api v0.21.2
-	k8s.io/apimachinery v0.21.2
-	k8s.io/client-go v0.21.2
-	sigs.k8s.io/controller-runtime v0.9.2
+	k8s.io/api v0.23.3
+	k8s.io/apimachinery v0.23.3
+	k8s.io/client-go v0.23.3
+	sigs.k8s.io/controller-runtime v0.11.0
```

Comment:
I remember having a problem with newer versions when receiving a webhook call. Definitely needs to double-check that in a real cluster, not just tests

Reply:
Checked with `v0.2.0-alpha` release in sandbox. Works fine.